### ISSUE TITLE
Running into issues with deploy

### ISSUE THREAD

**Issue by dbclkclk** [ryanbrainard/heroku-buildpack-phing#2]
Counting objects: 11, done.
Delta compression using up to 4 threads.
Compressing objects: 100% (8/8), done.
Writing objects: 100% (8/8), 1.20 KiB | 0 bytes/s, done.
Total 8 (delta 4), reused 0 (delta 0)
remote: Compressing source files... done.
remote: Building source:
remote:
remote: -----> Fetching set buildpack https://github.com/ryanbrainard/heroku-buildpack-phing.git... done
remote: -----> Phing app detected
remote: -----> Bundling Apache v2.2.22
remote: -----> Bundling PHP v5.3.15
remote: -----> Bundling Phing v2.4.6
remote: /tmp/build_9ae7720b7430775c06ee44da9319040d/php/bin/php: error while loading shared libraries: libssl.so.0.9.8: cannot open shared object file: No such file or directory
remote:
remote:  !     Push rejected, failed to compile Phing app
remote:
remote: Verifying deploy...
remote:
remote: !       Push rejected to teachpro.
remote:
To https://git.heroku.com/teachpro.git
 ! [remote rejected] master -> master (pre-receive hook declined)
error: failed to push some refs to 'https://git.heroku.com/teachpro.git'

**Comment by Feldhacker** [comment]
I'm hitting the same issue.  It seems the problem is caused because cedar-12 and heroku-16 no longer support ssl v0.9.8.  
See:  https://devcenter.heroku.com/articles/stack-packages
Looks like openssl needs to be upgraded.

**Comment by adamc-be** [comment]
As workbench.developerforce.com has just stopped working, I thought the right thing to do would be to spin up my own version on Heroku, as recommended.
Just run into this issue though.

Are there any solutions available to this yet?

**Comment by Jntshumaker** [comment]
@adamc-be I was able to deploy to heroku directly from github, this comment pointed me in the right direction: https://github.com/ryanbrainard/forceworkbench/issues/710#issuecomment-322555599

**Comment by Feldhacker** [comment]
Yeah, the docs are also old, so if you've encountered this error and are reading this thread you're heading down the wrong path.  Just clone the repo, customize the overrides file if desired, and push to Heroku -- workbench now seems to deploy just like any other Heroku app.  (Performance has been great, even on 1 dyno.)
